src/SprykerEco/Yves/Adyen/Form/AbstractSubForm.php

```diff
@@ -25,10 +25,18 @@ public function getProviderName(): string
     }
 
     /**
+     * @param array|null $options
+     *
      * @return \Symfony\Component\Validator\Constraint
      */
-    protected function createNotBlankConstraint(): Constraint
+    protected function createNotBlankConstraint(?array $options): Constraint
```

Comment:
You are saying this method accepts options, but then you take the message only out of it. It will make a user or this method confused. A much better thing to do here is really taking options as you promise:

```
return new NotBlank(
    array_merge(
        ['groups' => $this->getPropertyPath()], 
        $options
    )
);
```